### PR TITLE
make surface default cmaps same as in slice plotting

### PR DESCRIPTION
**Pull request by jeromedockes**
by default, `plot_stat_map` uses the `cold_hot` colormap. `plot_glass_brain`
does too, if the background is black, otherwise `cold_white_hot`.
`plot_surf_stat_map` uses `coolwarm`.

current default:
surf_stat_map_coolwarm

suggested default:

surf_stat_map_cold_hot

more importantly, `plot_roi` uses a rainbow colormap, namely
`gist_ncar`. `plot_surf_roi` uses `coolwarm`, which is not adapted for rois.

current default:

surf_roi_coolwarm

suggested default:

surf_roi_gist_ncar

I suggest we use the same defaults for surface plots as for slice plots.
note that the suggested defaults are the colormaps actually used in the
examples.

### CODE REVIEW

**Comment by bthirion** [comment]
Sounds good to me. Yes, we gain in consistency.

**Comment by GaelVaroquaux** [comment]
So much better! Thank you!

**Comment by jeromedockes** [comment]
> Hum, I realize that you haven't listed the changes in the whats_new. This would be useful. Can you do a new PR to add it?

sorry about that. see #1653

File: nilearn/plotting/surf_plotting.py [hunk 1]
```diff
@@ -483,7 +483,7 @@ def plot_surf_roi(surf_mesh, roi_map, bg_map=None,
                          'value for each vertex, or a list of Numpy arrays, '
                          'one array per ROI which contains indices of all '
                          'vertices included in that ROI')
-
+    vmin, vmax = np.min(roi_map), 1 + np.max(roi_map)
```
Comment (KamalakerDadi) [inline nilearn/plotting/surf_plotting.py]: Just for a point to be noted: Why is this needed ?

Comment (jeromedockes) [reply nilearn/plotting/surf_plotting.py]: otherwise if there is only one value (e.g. plotting a mask) it does not appear on the plot with some colormaps. for example try removing this line and running the plot_surf_stat_map.py example: the seed plot is empty

Comment (KamalakerDadi) [reply nilearn/plotting/surf_plotting.py]: But, we didn't caught this in the example before ?
